src/lando/api/views.py

```diff
@@ -305,3 +305,43 @@ def get(self, request: WSGIRequest, repo_name: str, number: int) -> JsonResponse
             target_repo, pull_request, request
         )
         return JsonResponse(warnings_and_blockers)
+
+
+class PullRequestTryPushAPIView(APIView):
+    @method_decorator(require_authenticated_user)
+    def post(
+        self, request: WSGIRequest, repo_name: str, pull_number: int
+    ) -> JsonResponse:
+        try:
+            try_repo = Repo.objects.get(name="try")
+        except Repo.DoesNotExist:
+            return JsonResponse({"errors": ["Try repo does not exist"]}, 500)
+
+        target_repo = Repo.objects.get(name=repo_name)
+        client = GitHubAPIClient(target_repo.url)
+        ldap_username = request.user.email
+        pull_request = client.build_pull_request(pull_number)
+
+        job = LandingJob.objects.create(
+            target_repo=target_repo,
+            is_handed_over=False,
+            handover_repo=try_repo,
+            requester_email=ldap_username,
+        )
```

Comment:
Do we need to specify `is_pull_request_job=True,` here?
```suggestion
        job = LandingJob.objects.create(
            target_repo=target_repo,
            is_handed_over=False,
            handover_repo=try_repo,
            requester_email=ldap_username,
            is_pull_request_job=True,
        )
```